packages/fast-usdc-deploy/test/snapshots/fast-usdc.test.ts.md

```diff
@@ -246,6 +246,81 @@ Generated by [AVA](https://avajs.dev).
       ],
     ]
 
+## deploy HEAD; update ChainHub for EVM/Solana
+
+> Under "published", the "fastUsdc.feeConfig" node is delegated to the fee configuration for Fast USDC with `customFeeConfigs`.
+> The example below illustrates the schema of the data published there.
+> 
+> See also board marshalling conventions (_to appear_).
+
+    [
+      [
+        'published.fastUsdc.feeConfig',
+        {
+          contractRate: {
+            denominator: {
+              brand: Object @Alleged: USDC brand {},
+              value: 10n,
+            },
+            numerator: {
+              brand: Object @Alleged: USDC brand {},
+              value: 2n,
+            },
+          },
+          'eip155:1': {
+            relay: {
+              brand: Object @Alleged: USDC brand {},
+              value: 1000000n,
+            },
+          },
+          'eip155:10': {
+            relay: {
+              brand: Object @Alleged: USDC brand {},
+              value: 500000n,
+            },
+          },
+          'eip155:137': {
+            relay: {
+              brand: Object @Alleged: USDC brand {},
+              value: 500000n,
+            },
+          },
+          'eip155:42161': {
```

Comment:
do we have human readable labels for these chains? it would be nice to publish them.
I guess it would be awkward, though

Reply:
We could use the `chainName` keys in `cctp-chain-info.js`. I think:
- it's readable enough to a sufficiently motivated person with access to https://chainlist.org/
- computers are a main consumer, and this should be clear enough to them